task/06-conditions-n-loops-tasks.js

```diff
@@ -209,7 +245,15 @@ function getIntervalString(a, b, isStartIncluded, isEndIncluded) {
  * 'noon' => 'noon'
  */
 function reverseString(str) {
-    throw new Error('Not implemented');
+    let i = 0;
+    let result = "";
+
+  while (i < str.length) {
+    let ch = str [str.length - i - 1];
+    result = result + ch;
+    i = i + 1;
+    }
+  return result;
```

Comment:
Try to convert string to array and use reverse method. It would be simpler and better